### PR TITLE
[OSDEV-2792] Add /jira-ticket skill and command — Story and Epic drafting + filing

### PR DESCRIPTION
**Pull request by TylerHeath1**
## Jira Ticket

[OSDEV-2792](https://opensupplyhub.atlassian.net/browse/OSDEV-2792)

---

## Summary of Changes

Adds the `/jira-ticket` skill and command — a coach that drafts, reviews, and files OS Hub Jira tickets (Stories and Epics) in the house style verified against recent OSDEV tickets.

Distinct from `/user-story` (which files Initiatives at the roadmap level): `/jira-ticket` creates **Stories** (sprint-ready, with Requirements and Given/When/Then Acceptance Criteria) and **Epics** (feature groups). Both work types share the words "user story" but mean different things in Jira's hierarchy — the skill calls this out upfront so the distinction stays clear.

Files added:
- `.agent/skills/jira-ticket-writer/SKILL.md` — canonical skill body
- `.claude/commands/jira-ticket.md` — thin command wrapper

The skill coaches with a vague-vs-specific push table (9 common vague phrasings + what to ask back) so tickets land specific enough for an engineer to pick up cold without coming back for clarification. It applies 4 tests live (specific persona / concrete capability / Requirements name specifics / AC covers happy path + edge case). Two checkpoints before filing: body approval, then metadata + final confirmation. Metadata is batched into 3 tiers (required-always-ask / optional-group / skip-unless-surfaced) so the prompt isn't 16 questions long. **Never files without a parent** — every ticket links to an Epic or Initiative.

---

## Type of Change

- [x] Documentation / tooling only

---

## Testing

Cannot run the full coaching + filing flow here — it'll be exercised on the next real ticket. Structural validation:
- [x] YAML frontmatter parses (`name`, `description`)
- [x] Skill references the correct Atlassian MCP fields (including required `customfield_10245` Funding Mechanism)
- [x] Story format matches verified OSDEV examples (OSDEV-2786, OSDEV-2778, OSDEV-2781)
- [x] Hands off to `/user-story` for Initiatives and `pr-description` for PR bodies

---

## Known TODO Items

- PR Description Writer skill (the in-place update of `.agent/skills/pr-description/SKILL.md` to match Max's new template from PR #1010) is the next skill to ship.
- When PR #1010 (the new `pull_request_template.md`) merges, the `pr-description` skill update lands too.

---

## Part of

The Common Dev Brain initiative — Epic [OSDEV-2719](https://opensupplyhub.atlassian.net/browse/OSDEV-2719).

Related PRs in the series: [#1011](https://github.com/opensupplyhub/open-supply-hub/pull/1011), [#1012](https://github.com/opensupplyhub/open-supply-hub/pull/1012), [#1013](https://github.com/opensupplyhub/open-supply-hub/pull/1013), [#1032](https://github.com/opensupplyhub/open-supply-hub/pull/1032), [#1035](https://github.com/opensupplyhub/open-supply-hub/pull/1035), [#1037](https://github.com/opensupplyhub/open-supply-hub/pull/1037), [#1038](https://github.com/opensupplyhub/open-supply-hub/pull/1038).

[OSDEV-2792]: https://opensupplyhub.atlassian.net/browse/OSDEV-2792?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[OSDEV-2719]: https://opensupplyhub.atlassian.net/browse/OSDEV-2719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .agent/skills/jira-ticket-writer/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+---
+
+## Story format (canonical)
+
+```
+**User Story:**
+
+* AS A [specific persona]
+* I WANT [the capability or change]
+* SO THAT [the outcome or benefit]
```
Comment (valencia-max) [inline .agent/skills/jira-ticket-writer/SKILL.md:28]: We already have a skill that create user stories, should we reference it here?

**Review by valencia-max — DISMISSED** [review]

File: .agent/skills/jira-ticket-writer/SKILL.md (hunk 1 above)
Comment (TylerHeath1) [reply .agent/skills/jira-ticket-writer/SKILL.md:28]: Good call — added a reference. The Story format now points the `AS A / I WANT / SO THAT` portion at `/user-story`: if you're starting from a raw customer ask, draft that part with `/user-story` first (it coaches persona → outcome → value and runs its four tests), then bring it here for Requirements + Acceptance Criteria. Reuse over duplication. (There was already a pointer in the Terminology section noting Initiatives are handled by `/user-story`; this adds it right at the format block where it's most useful.)

**Review by tana-bee — APPROVED** [review]
Looks good to me! Thanks for this work, Tyler!

**Review by valencia-max — APPROVED** [review]
🤖
